client/src/App.tsx

```diff
@@ -1,25 +1,23 @@
 import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import "./App.css";
 import { AuthContainer } from "./containers";
-import CssBaseline from "@mui/material/CssBaseline";
-import { createTheme, ThemeProvider } from "@mui/material/styles";
-
-const theme = createTheme({
-  palette: {
-    secondary: {
-      main: "#f4003d",
-    },
-  },
-});
 
 function App() {
   return (
-    <>
-      <ThemeProvider theme={theme}>
-        <CssBaseline />
-        <AuthContainer />
-      </ThemeProvider>
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<AuthContainer />} />
+        <Route
+          path="*"
+          element={
+            <main style={{ padding: "1rem" }}>
+              <p>There's nothing here!</p>
+            </main>
+          }
+        />
```

Comment:
Not sure if this is necessary. Having a `404 - not found` page is good, but it should be cleaner.

Reply:
Right, this is probably something that can be added to nginx later instead of here.